src/components/SelectionList/RadioListItem.js

```diff
@@ -1,51 +1,18 @@
 import React from 'react';
 import {View} from 'react-native';
-import CONST from '../../CONST';
-import PressableWithFeedback from '../Pressable/PressableWithFeedback';
 import styles from '../../styles/styles';
 import Text from '../Text';
-import Icon from '../Icon';
-import * as Expensicons from '../Icon/Expensicons';
-import themeColors from '../../styles/themes/default';
 import {radioListItemPropTypes} from './selectionListPropTypes';
 
-function RadioListItem({item, isFocused = false, isDisabled = false, onSelectRow}) {
+function RadioListItem({item, isFocused = false}) {
     return (
-        <PressableWithFeedback
-            onPress={() => onSelectRow(item)}
-            disabled={isDisabled}
-            accessibilityLabel={item.text}
-            accessibilityRole="button"
-            hoverDimmingValue={1}
-            hoverStyle={styles.hoveredComponentBG}
-            dataSet={{[CONST.SELECTION_SCRAPER_HIDDEN_ELEMENT]: true}}
-        >
-            <View style={[styles.flex1, styles.justifyContentBetween, styles.sidebarLinkInner, styles.optionRow, styles.userSelectNone, isFocused && styles.sidebarLinkActive]}>
-                <View style={[styles.flex1, styles.alignItemsStart]}>
-                    <Text style={[styles.optionDisplayName, isFocused ? styles.sidebarLinkActiveText : styles.sidebarLinkText, item.isSelected && styles.sidebarLinkTextBold]}>
-                        {item.text}
-                    </Text>
+        <View style={[styles.flex1, styles.alignItemsStart]}>
+            <Text style={[styles.optionDisplayName, isFocused ? styles.sidebarLinkActiveText : styles.sidebarLinkText, item.isSelected && styles.sidebarLinkTextBold]}>{item.text}</Text>
```

Comment:
We had to specify the `numberOfLines={1}` to show ellipsis if the text is too long instead of having it overflow